guides/doc-Release_Notes/master.adoc

```diff
@@ -9,7 +9,7 @@ ifdef::katello[]
 = Foreman {ProjectVersion} and Katello {KatelloVersion} Release Notes
 
 include::topics/foreman.adoc[leveloffset=+1]
-include::topics/katello.adoc[leveloffset=+1]
```

Comment:
Please keep this line in.

The idea is that we have a Foreman header, a Katello header and then individual releases in chronological order.

So it can be something like:

* Foreman intro
* Katello Intro
* Katello 4.6.1
* Foreman 3.4.2
* Foreman 3.4.1
* Katello 4.6.0
* Foreman 3.4.0

Reply:
@ekohl : Updated with the 2 changes.